.golangci.yml

```diff
@@ -0,0 +1,65 @@
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - copyloopvar
+    - dogsled
+    - dupl
+    - funlen
+    - gci
+    - gocognit
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goprintffuncname
+    - importas
+    - lll
+    - misspell
+    - nakedret
+    - nestif
+    - nolintlint
+    - staticcheck
+    - stylecheck
+    - unparam
+    - whitespace
+
+run:
+  timeout: 10m
+
+issues:
+  exclude-dirs:
+    - ^client/    # generated code
+    - ^pkg/api/   # generated code
```

Comment:
Recent commit by @PrasadG193 has generated mocks in a couple more `pkg` subdirectories.

Reply:
Done